src/components/_tooltip.scss

```diff
@@ -0,0 +1,68 @@
+@use "../design-tokens/color";
+@use "../design-tokens/typography";
+@use "../design-tokens/shadow";
+
+$tooltip-arrow-width: 5px;
+$tooltip-color: color.$color-grayDarker;
+
+@mixin Manifold__Tooltip {
+  position: relative;
+  max-width: 20rem;
+  min-height: 1rem;
+  padding: 8px 12px;
+  background: $tooltip-color;
+  color: color.$color-white;
+  border-radius: 4px;
+  text-align: center;
+  @include typography.Manifold__Typography__Caption;
+  line-height: 1.25;
+  box-shadow: shadow.$shadow-near;
+  cursor: default;
```

Comment:
Maybe contentious: this may need a `pointer-events: none;`, too. I think that as soon as you move off of the element, this should disappear. And it shouldn’t block clicks underneath if it’s invisible somewhere.